### PR TITLE
Maintainers file for ohai

### PR DESCRIPTION
**Pull request by thommay**
cc @chef/client-core

### CODE REVIEW

File: tasks/maintainers.rb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+# limitations under the License.
+#
+
+require 'rake'
+
+SOURCE = File.join(File.dirname(__FILE__), "..", "MAINTAINERS.toml")
+TARGET = File.join(File.dirname(__FILE__), "..", "MAINTAINERS.md")
+
+# The list of repositories that teams should own
+REPOSITORIES = ["chef/chef", "chef/chef-census", "chef/chef-repo",
```
Comment (mcquin) [inline tasks/maintainers.rb]: Should this just be "chef/ohai"?

File: tasks/maintainers.rb [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  def github
+    @github ||= Octokit::Client.new(:netrc => true)
+  end
+
+  def source
+    @source ||= Tomlrb.load_file SOURCE
+  end
+
+  def teams
+    @teams ||= {"client-maintainers" => {"title" => "Client Maintainers"}}
```
Comment (btm) [inline tasks/maintainers.rb]: Client maintainers?

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 *.sw?
 Gemfile.lock
+MAINTAINERS.md
```
Comment (btm) [inline .gitignore]: Don't we want this because it's the readable version?

**Comment by btm** [comment]
Is the maintainer list just the core list? Is that appropriate? Should we ensure everyone has been an ohai contributor too?

**Comment by jaymzh** [comment]
I think client-core is the right call....

**Comment by lamont-granquist** [comment]
Its effectively how we operate anyway, same with the core mixlib's that chef depends on.

**Comment by tas50** [comment]
Closing this in favor of a PR against chef/chef

**Pull request closed without merging**